src/grammar.ne

```diff
@@ -37,8 +42,16 @@ combinator ->
 selector -> selectorBody {% d => ({type: 'selector', body: d[0]}) %}
 
 selectorBody ->
-    typeSelector:? idSelector:? classSelector:* attributeValueSelector:* attributePresenceSelector:* pseudoClassSelector:* pseudoElementSelector:? {% (d, i, reject) => { const selectors = flatten(d); if (!selectors.length) return reject; return selectors; } %}
-  | universalSelector idSelector:? classSelector:* attributeValueSelector:* attributePresenceSelector:* pseudoClassSelector:* pseudoElementSelector:? {% flatten %}
+    typeSelector:? simpleSelector:* {% (d, i, reject) => { const selectors = flatten(d); if (!selectors.length) return reject; return selectors; } %}
+  | universalSelector simpleSelector:* {% flatten %}
+
+simpleSelector ->
+  idSelector
+| classSelector
+| attributeValueSelector
+| attributePresenceSelector
+| pseudoClassSelector
+| pseudoElementSelector
```

Comment:
This also includes changes made in my pending PR to scalpel: https://github.com/gajus/scalpel/pull/7